config.yaml

```diff
@@ -50,6 +50,7 @@ options:
 
   release:
     type: string
+    default: "v3.9.0"
```

Comment:
Hmm... i'm not sure i like defaulting here.  You could just leave v3.9.0 in the `version` file right?  That should update the defacto-default without changing the charm config right?

What's the intent maybe i'm missing it

Reply:
@addyess i felt like `version` was too easy to accidentally bump.  any time you update.py to bring in new manifests, `version` is reset.  i liked config as a more explicit place to say "this charm is known to work with this version" as well as being able to unset the config to say "just give me the latest".

i don't feel too strongly either way, but that was my train of thought.  if you'd prefer using `version` for the default config, i can get behind that.

Reply:
nah, that's a good enough reason for me!  Someone can config this value to adjust releases!
👍🏻